### ISSUE TITLE
Support .NET Core 3.1.201 (and .NET 5)

### ISSUE THREAD

**Issue by aalmada** [unoplatform/Uno.SourceGeneration#117]
## I'm submitting a...

  
Feature request

## Current behavior

When using .NET 5.0 Preview 1, results in the error:

`/Users/antao.almada/.nuget/packages/uno.sourcegenerationtasks/2.0.0-dev.308/build/netstandard1.0/Uno.SourceGenerationTasks.targets(127,2): error : Exec format error [/Users/antao.almada/Projects/NetFabric.Hyperlinq/NetFabric.Hyperlinq/NetFabric.Hyperlinq.csproj]`

## Minimal reproduction of the problem with instructions

Install [.NET 5.0 Preview 1](https://dotnet.microsoft.com/download/dotnet-core/5.0) and compile the generator project.

## Environment

```
Nuget Package: 
`Uno.SourceGeneration`

Package Version(s): 
`2.0.0-dev.308`

Affected platform(s):
- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] Windows
- [ x] Build tasks

Visual Studio
- [ ] 2017 (version: )
- [ ] 2017 Preview (version: )
- [ ] for Mac (version: )
- [x] .NET CLI

Relevant plugins
- [ ] Resharper (version: )
```

**Comment by jeromelaban** [comment]
That was fast ;) Thanks for letting us know about it. Source generation will definitely have to run there, and it should not be too much of a stretch.

**Comment by aalmada** [comment]
@jeromelaban I'm getting the same error with [.NET Core 3.1.201](https://dotnet.microsoft.com/download/dotnet-core/3.1). Have you experienced it already?

**Comment by aalmada** [comment]
@jeromelaban Is this fix available in a NuGet package? I'm using 2.0.0-dev.308 from the private source.

**Comment by jeromelaban** [comment]
It's available here: https://www.nuget.org/packages/Uno.SourceGenerationTasks/2.0.0-dev.16
